index.html

```diff
@@ -1,48 +1,297 @@
+
+
+                <div class=" item4 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                <div class="item4"><button class="see-project">See project</button></div>
+                
+                
+
+
+
+            </section>
+        </div>
+
+        <div>
+                <section id="works3" class="container5">
+
+
+                    <div class="item5"><img alt="work3" src="img/work3.png"> </div>
+
+                    
+                                
+                    <div class="item5"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                    <div class=" item5 btn-group">
+                        <button class="languages"> Ruby on rails</button>
+                        <button class="languages"> css</button>
+                        <button class="languages"> JavaScript</button>
+                        <button class="languages"> html</button>
+                    </div>
+
+                    
+                    <div  class="item5"><button class="see-project">See project</button></div>
+                    
+                
+
+
+
+                </section>
+        </div>
+
+        <div>
+            <section id="works4" class="container6">
+
+
+                <div class="item6"> <img alt="work4" src="img/work4.jpg"></div>
+
+                    
+                                
+                <div  class="item6"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item6 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                <div class="item6"><button class="see-project">See project</button></div>
+            
+            
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works5" class="container7">
+
+
+                <div class="item7"> <img alt="work5" src="img/work5.png"> </div>
+
+                    
+                                
+                <div class="item7"> <h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item7 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                <div class="item7"><button class="see-project">See project</button></div>
+            
```

Comment:
- [ ] The button (e.g., "See project") is a link a or button tag nice it is a button, and  you followed the button styling: Active, Hover and Pressed, however you didn't add the `hover color` as well as add `cursor: pointer`